### PR TITLE
Write documentation in Markdown using MyST

### PR DESCRIPTION
**Pull request by cauliyang**
## Reason

I am a fresh developer. This template is awesome and helps me a lot. Also, related blogs enable me to develop my own library and know more about how to build a library. However, I found that readme is in RST format, which may cause some inconvenience when I would like to insert an image and change size or position. I also try using `.. image::` to change image position but it does not work in Github. What is more, `.. raw::` is not supported by `Pypi` so that I cannot use HTML style. I know **myst** may fix the problem. But I only would like to change the readme file. Hence, I change some code in order to provide an option to choose what is the format of the readme.  I am sure that there are some people who have the same needs as me.

I also test current code and run all `nox` sessions, there is no error. However, I do not know my idea is good and whether the code is compatible totally. :)

### CODE REVIEW

**Review by cjolowicz — COMMENTED** [review]
Thank you for this contribution! Adding MyST to this project template has been on my mind for a while. So it's good to see this spelled out for the README. Nice work!

Unfortunately, creating a template variable to control the format of the README is not an approach I want to take for this template. There are two reasons for this:

1. I'm very reluctant to add configuration options to the template, generally. I like there to be only one way to use this template, where possible. If only because that keeps it more maintainable and testable for myself.
2. When we switch documentation to Markdown, I would prefer all of it to switch, not just the README. Sadly, that's currently not possible at all, because docstrings still have to be written in RST. (I *might* compromise on this point though.)

So that means that I cannot accept this PR. I would consider a PR that works on the much larger task of switching everything to MyST. If you're willing to go down this road, let's talk it over carefully though beforehand.

I hope you understand, and thanks again for taking the time to work on this!

**Comment by cauliyang** [comment]
Many thanks for your review! Myst does be a good tool and can use the power of rst and markdown at the same time.  I only change the readme due to two reasons:

1. Learning a new tool may increase the time cost and may raise some unexpected errors. Some people may want their projects to look better by using relatively short time.  

2. the readme of the project can be customized powered by the property of markdown. As for other document files(usage, reference, etc), they still use `rst` because docstring is written by `rst` style.  This also considers the time cost.

However, I totally agree with your options as adding a new configuration may break the habit of using it. That also damages maintainable. Also, my change is not "clean" and perfect.  

I have a question about what do you mean by switching everything to MyST, including docstring? I see that MyST can use [`autodoc`](https://myst-parser.readthedocs.io/en/latest/sphinx/use.html#howto-autodoc).  This may allow us to write docstring with rst or others(google style, numpy) and write document files(reference, usage, index) with markdown.  [This issue](https://github.com/executablebooks/MyST-Parser/issues/228) of MyST talks about writing docstrings with markdown. However, I think writing docstring with rst or others(google style, numpy may be good and looks better. 

I am very willing to contribute something to this repo as it helps me a lot!  If you think my option is not on the right track please recorrect me! I like learning new things! :)

**Comment by cjolowicz** [comment]
> I have a question about what you mean by "switching everything to MyST", including docstring?

I mean converting all reStructuredText documentation to Markdown, using MyST. Docstrings would have to stay in reStructuredText for now. But I guess it's ok.

**Comment by cauliyang** [comment]
Thanks for your response , and that sounds so great 👍 . I will try to do that and hope to help the repo.

**Comment by cjolowicz** [comment]
Sure, thanks for helping out! Let me know if you have any questions. Feel free to open a draft PR anytime you feel like it.

**Comment by cjolowicz** [comment]
By the way, you can also open your PR in the instance repo: https://github.com/cjolowicz/cookiecutter-hypermodern-python-instance

It will be imported into the project template on merge.

**Comment by cauliyang** [comment]
Hi @cjolowicz,

I have finished the changing for files under "docs/" as well as README.md, CONTRIBUTING.md, CODE_OF_CONDUCT.md. 
Also, the license files are converted to md.

I have tested the commit,  all looks good except one:

When I test `sphinx build doc`

image

[this issue](https://github.com/readthedocs/recommonmark/issues/130) said that this may be fixed when documents are hosted in the cloud(readthedocs) instead of the local environment.

I will also create an instance and open PR to the reop later if there is no other problem: the instance repo: https://github.com/cjolowicz/cookiecutter-hypermodern-python-instance

What do you think?

**Comment by pauleveritt** [comment]
Hi @cauliyang I'm very interested in this topic. I talked with @cjolowicz and offered to do the review. Is that ok? He gave me some of the things he's looking for before merging.

**Comment by cauliyang** [comment]
Hi @pauleveritt , 

Sorry for the late reply, and I now try to create a test instance to check if `myst reference target not found` still exists when uploading to `readthedoc`.   I am very happy to talk with you about this change^^ 

What do you think about what I submit now? Is there something wrong?

**Comment by cauliyang** [comment]
Hi @pauleveritt, 

I have fixed the error about `myst target` according to changes of @paw-lu 's repo. Now the document system of `md` works well. 

Do you think where I can improve to make that better?

**Comment by pauleveritt** [comment]
Sorry, I should have put in the bullet points from @cjolowicz as soon as we talked last week, rather than make you wait. Here were his points:

- **Include original license text**. Don't use the markdown-ification. For legal reasons, better to keep originals. Also lets us track "upstream" more closely, if it changes.
- **Code of Conduct**. He said there is a Markdown version "upstream", rather than us doing the rst->md ourselves. (Again, tracking upstream.)
- **Links for all sites**. He said "Have the links work for all sites (GitHub, PyPI, etc.) as the relative links break." I kind of know what he means, but not exactly. 😀

None of this prevents me from starting a review, though.

**Comment by paw-lu** [comment]
> I have fixed the error about myst target according to changes of @paw-lu 's repo. Now the document system of md works well.

I'm glad it helped! Thanks a bunch @cauliyang for implimenting it!

> Do you think where I can improve to make that better?

I'll let @pauleveritt take over the review to limit the noise here, but have you looked at my other suggestions above? Highlights:

1. Revert syntax from `bash` → `console`
2. Remove prepending whitespace from console commands `    $ nox --list-sessions` → `$ nox --list-sessions`
3. Change empty `github-only` code blocks to comments ```` ```github-only```` → `% github-only`
4. Remove comments and placeholders (`#temp` in `.gitignore`, placeholder links)
5. Remove uneeded empty brackets for hyperlinks `[link text][]` → `[link text]`

---

Again, thanks a lot for helping out here!

**Comment by paw-lu** [comment]
> - Links for all sites. He said "Have the links work for all sites (GitHub, PyPI, etc.) as the relative links break." I kind of know what he means, but not exactly. 😀

Just to be clear here (for everyone), a relative link like this:

```markdown
Check out [CONTRIBUTING.md](CONTRIBUTING.md) for general contribution guidelines.
```

would work correctly when rendining on GitHub, but would not work on the rendered the README on PyPI. I even have an example of one I never got around to fixing. Check out the `CONTRIBUTING.md` link on the very bottom of both of these:

- [PyPI](https://pypi.org/project/dressup/)
- [GitHub](https://github.com/paw-lu/dressup)

It only works properly on GitHub.

We might run into related issues also when rendering the `README.md` content on Read the Docs, which is why @cjolowicz originally used `end-before: github-only` comments to swap out links to prevent breaking.

So we have three main locations our pages render:

1. GitHub
2. PyPI
3. Read the Docs

And want to make sure hyperlinks work as expected across all.

---

Also while you're here, I really enjoyed your static site talk @pauleveritt!

**Comment by cauliyang** [comment]
Hi @pauleveritt, 

Thanks for your recommendation, and I will try to follow the rules you mentioned here. Later, I will submit newest change.
😄

**Comment by cauliyang** [comment]
Hi @paw-lu , 

it is so regretted that I miss your suggestions before, and I will go to read that carefully. Big thanks for your valuable review, and I will change these you mentioned as soon as possible. 

I am happy to contribute this project and hope the project better 🚀

**Comment by paw-lu** [comment]
> - Include original license text. Don't use the markdown-ification. For legal reasons, better to keep originals. Also lets us track "upstream" more closely, if it changes.

To stay in sync with this template I also attempted to impliment this in a project generated from this repo. See PR if it helps:

- https://github.com/paw-lu/nbpreview/pull/394/

**Comment by pauleveritt** [comment]
Thanks for the kind words @paw-lu and hope to see you more often around in this repo. Do you have time in 2022 to join in?

@cauliyang Do you think you can also handle the list from @paw-lu ?

**Comment by cauliyang** [comment]
@pauleveritt, I will handle the list from @paw-lu as well. Hope we get a nice result!

**Comment by cauliyang** [comment]
Finished:


@pauleveritt 
- [x] 1. Include original license text
- [x] 2. Code of Conduct
- [x] 3. Links for all sites.
@paw-lu 
- [x] 4. Revert syntax from bash → console
- [x] 5. Remove prepending whitespace from console commands
- [x] 6. Change empty github-only code blocks to comments github-only → % github-only
- [x] 7. Remove comments and placeholders (#temp in .gitignore, placeholder links)
- [x] 8. Remove empty brackets for hyperlinks [link text][] → [link text]

File: {{cookiecutter.project_name}}/docs/license.md
Comment (pauleveritt) [inline {{cookiecutter.project_name}}/docs/license.md:5]: Small detail...is this required or can we just do it all on one line, no YAML options passed?

File: {{cookiecutter.project_name}}/CODE_OF_CONDUCT.md
Comment (pauleveritt) [inline {{cookiecutter.project_name}}/CODE_OF_CONDUCT.md]: Is the extra `[]` needed here (and in the next line)? I believe an earlier change flattened those.

File: {{cookiecutter.project_name}}/CODE_OF_CONDUCT.md
Comment (pauleveritt) [inline {{cookiecutter.project_name}}/CODE_OF_CONDUCT.md]: Just to be careful, let's get this period to the previous line (IMO).

Hmm, if this is an upstream version, then ignore this comment and the previous one.

File: {{cookiecutter.project_name}}/CONTRIBUTING.md
Comment (pauleveritt) [inline {{cookiecutter.project_name}}/CONTRIBUTING.md]: Can all the `[]` be removed here? I believe I saw this in previous commits.

File: {{cookiecutter.project_name}}/CONTRIBUTING.md
Comment (pauleveritt) [inline {{cookiecutter.project_name}}/CONTRIBUTING.md]: Per @paw-lu should you remove the prepending white space in the various places in this doc?

File: {{cookiecutter.project_name}}/README.md
Comment (pauleveritt) [inline {{cookiecutter.project_name}}/README.md]: Same thing in this docs with `[]`

**Comment by pauleveritt** [comment]
@cauliyang Hah, I see with `.idea` in `.gitignore` that you're using PyCharm. Thanks. 😀

Sorry for all the noise, I should have done a review instead of individual comments. Also, I wonder if I was looking at wrong revisions on all the `[]` comments.

All looks good, I ran the cookiecutter and inspected the results. @cjolowicz I suggest merging this. We can then discuss any other items as new tasks.

File: {{cookiecutter.project_name}}/docs/license.md
Comment (cauliyang) [reply {{cookiecutter.project_name}}/docs/license.md:5]: For this case, we can rewrite it as:

\```{literalinclude} ../LICENSE 
\```

But we can't just do it all no one line, like this \```{literalinclude} ../LICENSE\```  it will not render ^ ^


Also, yaml options can be replaced with: 


\```{literalinclude} ../LICENSE
:language: none
\```

File: {{cookiecutter.project_name}}/CODE_OF_CONDUCT.md
Comment (cauliyang) [reply {{cookiecutter.project_name}}/CODE_OF_CONDUCT.md]: Thanks for your review again.

In fact, I have adopted the upstream version, and the current content is: 


## Attribution

This Code of Conduct is adapted from the [Contributor Covenant][homepage],
version 2.0, available at
[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].


you can check [this](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/CODE_OF_CONDUCT.md)

File: {{cookiecutter.project_name}}/CODE_OF_CONDUCT.md
Comment (cauliyang) [reply {{cookiecutter.project_name}}/CODE_OF_CONDUCT.md]: Same as mentioned above,

current content in my forked repo:

## Attribution

This Code of Conduct is adapted from the [Contributor Covenant][homepage],
version 2.0, available at
[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].

Community Impact Guidelines were inspired by 
[Mozilla's code of conduct enforcement ladder][Mozilla CoC].

You can check [this](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/CODE_OF_CONDUCT.md) 


Maybe you see outdated file^ ^

File: {{cookiecutter.project_name}}/CONTRIBUTING.md
Comment (cauliyang) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md]: Same as mentioned above 

I have changed this kind of error before. 

[this](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/CONTRIBUTING.md)

File: {{cookiecutter.project_name}}/CONTRIBUTING.md
Comment (cauliyang) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md]: Same as mentioned above. 

Current content:

Install the package with development requirements:

\```console
$ poetry install
\```

File: {{cookiecutter.project_name}}/README.md
Comment (cauliyang) [reply {{cookiecutter.project_name}}/README.md]: This file is also outdated. [current content](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/README.md)

**Comment by cauliyang** [comment]
@pauleveritt, Really appreciate your review. Big respect to you😄! aHa, I am a big fan of JetBrains, and I use Pycharm, Clion heavily in the remote HPC. They have very cool features to support that. I also try DataShell and Gateway. DataShell currently may not have advantages over its alternatives. Also, I notice that fleet, and I am looking forward to it.

By the way, thanks for your team or company supporting student license 🚀 .

**Comment by cauliyang** [comment]
@pauleveritt, 

I change the layout of the badge, which means they are centered instead of left. 

That may be a "big change" so I would like to make you notice that

**Comment by paw-lu** [comment]
> Thanks for the kind words @paw-lu and hope to see you more often around in this repo. Do you have time in 2022 to join in?

Yeah of course! I've really enjoyed this project, so happy to help @pauleveritt

**Comment by paw-lu** [comment]
@cauliyang really good work here!

> Do you think there is something error?

[There are just a few unaddressed suggestions left over.](https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1089/files) They're all super minor. One is (I think) a lefover comment and the other are formatting errors that would cause the CI to fail.

**Comment by paw-lu** [comment]
Is b91d8c3e4ada4ba24dd5023b445597d7c1caee41 mean to be a copy of https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1101?

Could we rebase onto main instead of duplicating the diff[^1]?

[^1]: Quick way to do it: `git fetch origin; git reset --hard ee0a319; git rebase --interactive origin/main`, then adjust any conflicts.

**Comment by cauliyang** [comment]
@paw-lu, Thanks for your valuable recommendation! Could you give more information, and I will continue to improve that. 

https://github.com/cjolowicz/cookiecutter-hypermodern-python/commit/b91d8c3e4ada4ba24dd5023b445597d7c1caee41 does a copy of #1101 

The reason I add is to make sure that cookiecutter works to test the template.

**Comment by cauliyang** [comment]
Hi @paw-lu, I think you may check the outdated version, and I do not know why you and @pauleveritt see this version. I have changed all issues you mentioned above several days ago. See [my repo](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/docs/contributing.md)

See [readme](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/README.md)

**Comment by paw-lu** [comment]
> Thanks for your valuable recommendation! Could you give more information, and I will continue to improve that.

Realized I never published the review @cauliyang 😅. It's published now, I've marked most of them as `outdated`—as you've addressed almost all of them. But there's a few leftover! Again, most are only really minor things left to make sure you pass the CI.

Specifically:

- https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1089#discussion_r785562748
- https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1089#discussion_r786204801
- https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1089#discussion_r787327454
- https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1089#discussion_r790171304

> b91d8c3 does be a copy of #1101
> 
> The reason I add is to make sure that cookiecutter works to test the template.

Yeah, that's a really good idea! Instead of duplicating the changes from #1101 in b91d8c3, we can instead just directly use b91d8c3 by running `git rebase`. It will first update your branch to look like `main`, then move your changes on top of it. I've tried to sketch out the basic steps to accomplish that in the footnote of https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1089#issuecomment-1019341576. This way we won't have two commits that actually are doing the same thing, and that when we look at the [PR diff](https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1089/files) it will only show the _new_ changes we've made here—not ones made in other PRs like #1101.

But this second point is not as important, we can always rebase later when it's time to merge!

**Comment by paw-lu** [comment]
> Hi @paw-lu, I think you may check the outdated version, and I do not know why you and @pauleveritt see this version. I have changed all issues you mentioned above several days ago. See my repo

Yeah that's just because I wrote this review last week 😅! I've (tried) to mark all of the one's you've fixed as `outdated`—so ignore those. I've tried to cut down on the noise by highlighting the outstanding ones in https://github.com/cjolowicz/cookiecutter-hypermodern-python/pull/1089#issuecomment-1019357797.

File: {{cookiecutter.project_name}}/README.md
Comment (cauliyang) [reply {{cookiecutter.project_name}}/README.md:70]: We can remove that or keep that. It is just a comment to indicate that links.

**Comment by cauliyang** [comment]
Oh, I see. Last week, you asked me have checked your suggestions?.  I am actually confused about where I should see that ^ ^. 

For now, there is still a blank line in `.gitignore`.  I will fix that now. 

For centering the badge and `[//]: #link`, I think we can remove or keep that. Both options are alright.

**Comment by cauliyang** [comment]
@paw-lu, Many thanks for @pauleveritt  and @paw-lu . Respect to you two 👏 

You help me a lot to improve my change

File: {{cookiecutter.project_name}}/docs/codeofconduct.md
Comment (cauliyang) [reply {{cookiecutter.project_name}}/docs/codeofconduct.md:2]: We need to leave a blank line there, right?

**Comment by cauliyang** [comment]
Finished:

- [x] remove a blank line in `.gitignore`
- [x] add a blank line to pass the prettier formatting pre-commit check
- [x] remove comment in markdown : ` [//]: #link`
- [x] center badge

@paw-lu and @pauleveritt, is there something I can improve?

**Comment by cjolowicz** [comment]
Thanks everybody for the great work here 💚 

@cauliyang There seem to be some merge conflicts. Could you rebase your branch on main? (Or merge main into your branch, if you prefer.) Let me know if you have any questions about that.

**Comment by staticdev** [comment]
I am just watching this PR with interest and also want to congratulate @cauliyang and everyone here.
Also want to mention @pauleveritt I am a big fan, watch hours and hours of your videos =)

**Comment by cauliyang** [comment]
Hi @cjolowicz , 

I have merged main into my branch, and there are no conflicts now.  Please feel free to check that ^ ^ 

Again, thanks for @paw-lu and @pauleveritt s' great help and review!

**Comment by pauleveritt** [comment]
Thanks @cauliyang for the work and @paw-lu for most of the useful input/

@staticdev Thanks for the kind word. I'm planning a big expansion this year on that front, would be interested in your input. Email me first dot last at company dot com.

**Comment by staticdev** [comment]
> Hi @cjolowicz ,
> 
> I have merged main into my branch, and there are no conflicts now. Please feel free to check that ^ ^
> 
> Again, thanks for @paw-lu and @pauleveritt s' great help and review!

@cauliyang pre-commit is still complaining.. try to run `poetry install` and then `nox -s pre-commit`, it will autofix the issues.

**Review by staticdev — CHANGES_REQUESTED** [review]
@cauliyang thanks once again for the PR. There is still some tricky pre-commit issues generating errors on Github Actions. To fix it, you need to generate a project using your repo:

`cookiecutter --no-input gh:cauliyang/cookiecutter-hypermodern-python` this generates a project with the defaults. Then if you run ```
poetry install
nox -s pre-commit
```

You will see these generated files are missing the end of file:
- CODE_OF_CONDUCT.md
- README.md
- docs/license.md
- .gitignore
- docs/requirements.txt
- docs/contributing.md
- docs/codeofconduct.md

Also there are some trailing whitespaces on:
Fixing CODE_OF_CONDUCT.md
Fixing docs/reference.md

**Comment by cauliyang** [comment]
Thanks, @staticdev, I have fixed all the problems preventing pre-commit passing the check.

@pauleveritt, I like Compare of Pycharm ❤️

**Review by staticdev — APPROVED** [review]

**Review by paw-lu — APPROVED** [review]

File: {{cookiecutter.project_name}}/CONTRIBUTING.md
Comment (paw-lu) [inline {{cookiecutter.project_name}}/CONTRIBUTING.md:113]: This was my fault! I suggested this change, but it seems GitHub Flavored markdown does not support `%` commenting. My bad! We'll need to switch to something that GitHub, PyPI, and MyST respect.

Comment (paw-lu) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md:113]: Seems `<!-- -->` is [the officially documented syntax for GitHub Flavored Markdown comments](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#hiding-content-with-comments). But oes this work on MyST and PyPI?

Comment (cauliyang) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md:113]: I know that GitHub cannot render `%` comments, and before I try to use 

\```github-only
\```

But this will level a blank:
```github-only
```

I will try `<!-- -->` later to see if it works for GitHub, PyPI, and myst

Comment (cjolowicz) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md:113]: Using `<!-- -->` should work. But you'll probably need to change the include directives in `contributing.rst` and `index.rst` to match the entire comment (`<!-- github-only -->`), not just the `github-only` part. Otherwise, docutils will include everything up to the `<!--`, turning the remaining document into a comment.

Comment (cauliyang) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md:113]: Got it! Thanks @cjolowicz

Comment (eyllanesc) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md:113]: Use    `<!-- github-only -->` in README.md and 

index.md

```markdown
```{include} ../README.md
---
end-before: <!-- github-only -->
---
```

Comment (cauliyang) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md]: Changed

File: {{cookiecutter.project_name}}/CONTRIBUTING.md
Comment (cjolowicz) [inline {{cookiecutter.project_name}}/CONTRIBUTING.md:19]: This heading needs to be second-level. Same for all headings after it.

```suggestion
## How to report a bug
```

This is important so the subheadings don't get displayed in the navigation sidebar in Sphinx.

Comment (cauliyang) [reply {{cookiecutter.project_name}}/CONTRIBUTING.md]: fixed: ab9b60411355c0504080d280084c96082862bf72

Tested result:
image

**Comment by cauliyang** [comment]
Hi @cjolowicz , I notice that there exist conflicts with  version of `sphnix-click`: 

```
<<<<<<< main
sphinx-click==3.0.2
myst_parser==0.16.1
=======
sphinx-click==3.1.0
>>>>>>> main
```
I can fix that as below:

```
sphinx-click==3.1.0
myst_parser==0.16.1
```
Does that work? I am concerned that I miss something

**Comment by eyllanesc** [comment]
@cauliyang It should work, it seems that while you were implementing the PR the sphinx-click tool was updated.

File: {{cookiecutter.project_name}}/README.md
Comment (cjolowicz) [inline {{cookiecutter.project_name}}/README.md:3]: Thanks for the update @cauliyang 

These images need to be templatized, there should be no "hypermodern-python".

Also it seems that the alt text is not the same as in the .rst version.

Thanks!

Comment (eyllanesc) [reply {{cookiecutter.project_name}}/README.md:3]: @cauliyang @cjolowicz 

It was not only in README.md but in other files, I have created a PR adding those changes.

Comment (cauliyang) [reply {{cookiecutter.project_name}}/README.md]: Hi @cjolowicz, 

Thanks for your review 😄 ! 

That is my fault due to my carelessness.  I have fixed that and tried to use an elegant way to change the alt text.   I tested GitHub, PyPI, readthedocs as well. All works well.

Comment (cauliyang) [reply {{cookiecutter.project_name}}/README.md]: Thanks for your help! @eyllanesc 


You help me identify what is wrong. But you change 'title' instead of the alt text.  Anyway, Thanks for your time and help:heart: !

File: {{cookiecutter.project_name}}/docs/requirements.txt
Comment (staticdev) [inline {{cookiecutter.project_name}}/docs/requirements.txt:1]: This should not be using template.. could you fix this also? It should be furo==2022.1.2

**Review by staticdev — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_name}}/docs/requirements.txt
Comment (cauliyang) [reply {{cookiecutter.project_name}}/docs/requirements.txt:1]: Thanks for your review! @staticdev , you can check [here](https://github.com/cjolowicz/cookiecutter-hypermodern-python/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/docs/requirements.txt). 
This is consistent with the current version.

Comment (staticdev) [reply {{cookiecutter.project_name}}/docs/requirements.txt:1]: I know, I just noticed since you changed this file. But this is not intended, right @cjolowicz ?

Comment (cjolowicz) [reply {{cookiecutter.project_name}}/docs/requirements.txt:1]: right, this is a version number and should not be templated

Comment (cauliyang) [reply {{cookiecutter.project_name}}/docs/requirements.txt]: Thanks, @staticdev, and @cjolowicz! It is fixed.

File: {{cookiecutter.project_name}}/README.md
Comment (cjolowicz) [inline {{cookiecutter.project_name}}/README.md:74]: This cannot be a relative link. It works on GitHub, but not on the PyPI project page.

```suggestion
[contributor guide]: [CONTRIBUTING.md](https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/blob/main/CONTRIBUTING.md)
```

Apart from this, this is ready to be merged 💯

Comment (cauliyang) [reply {{cookiecutter.project_name}}/README.md]: Fixed that! @cjolowicz

**Review by cjolowicz — APPROVED** [review]
Huge thanks to @cauliyang and everybody who helped here @pauleveritt @paw-lu @staticdev @eyllanesc

**Pull request merged**
